EBH-3338/medicover.py

```
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
```

Ruff (0.8.2)

Shebang is present but file is not executable

(EXE001)